archinstall/lib/disk/helpers.py

```diff
@@ -221,10 +221,8 @@ def all_blockdevices(mappers=False, partitions=False, error=False) -> Dict[str,
 		device_path = f"/dev/{pathlib.Path(block_device).readlink().name}"
 		try:
 			information = blkid(f'blkid -p -o export {device_path}')
-
-		# TODO: No idea why F841 is raised here:
-		except SysCallError as error: # noqa: F841
-			if error.exit_code in (512, 2):
+		except SysCallError as ex:
```

Comment:
`as error` was overridding the function argument with the same name

Reply:
Oh snap! I think we don't use the `error` parameter anymore.
I can't find any reference to it in this function at least.
Perhaps we can remove the parameter?